start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
```

Comment:
Interested in a **Patlet** in this pattern?

Reply:
absolutely!